### ISSUE TITLE
Error with 3.0.5 release

### ISSUE THREAD

**Issue by andersforsell** [vaadin-component-factory/selection-grid-flow#56]
Hi,
After upgrading to the 3.0.5 release I get this error in the browser after selecting an item:

```
generated-flow-imports-CuNoyU_Z.js:13765  Uncaught TypeError: this.$server.selectRangeOnlyOnClick is not a function
    at VcfSelectionGridElement.<anonymous> (generated-flow-imports-CuNoyU_Z.js:13765:24)
    at later (generated-flow-imports-CuNoyU_Z.js:13715:26)
(anonymous) @ generated-flow-imports-CuNoyU_Z.js:13765
later @ generated-flow-imports-CuNoyU_Z.js:13715
setTimeout (async)
_debounce @ generated-flow-imports-CuNoyU_Z.js:13719
_selectionGridSelectRowWithItem @ generated-flow-imports-CuNoyU_Z.js:13764
_selectionGridSelectRow @ generated-flow-imports-CuNoyU_Z.js:13706
_click @ generated-flow-imports-CuNoyU_Z.js:13925
```

**Comment by pellmann** [comment]
I have the same problem and it is still there with 3.0.6.

**Comment by andersforsell** [comment]
Good point, I see now that the new `@ClientCallable` method  `selectRangeOnlyOnClick` was not added to SelectionTreeGrid.

**Comment by mlopezFC** [comment]
Thanks for reporting this! .. I will take a look and let you know.

**Comment by paodb** [comment]
Fixed in version [3.0.7](https://github.com/vaadin-component-factory/selection-grid-flow/releases/tag/v3.0.7).
